commcare_connect/reports/helpers.py

```diff
@@ -78,6 +80,7 @@ def get_table_data_for_year_month(
         date_paid__lt=end_date,
     )
 
+    connectid_user_count_data = fetch_user_counts()
```

Comment:
Doesn't seem ideal that we are fetching all months data but we need only the selected month data, so may be pass a month param to the API?